modules/apps/foundation/address-formatter/address-formatter-api/src/main/java/com/liferay/address/formatter/BaseAddressFormatter.java

```diff
@@ -37,7 +43,7 @@ public String getCountry(Address address) {
 		Country country = countryService.fetchCountry(countryId);
 
 		if (country != null) {
-			return country.getName();
+			return country.getName(getUserLocale());
```

Comment:
You are right, I misread the condition and misinterpreted the change.

You can ignore this comment completely :)